src/main/java/com/mitchej123/hodgepodge/util/StringUtil.java

```diff
@@ -0,0 +1,43 @@
+package com.mitchej123.hodgepodge.util;
+
+// Methods stolen from FontRenderer
```

Comment:
Do not copy MC code. Make the FontRenderer methods public with an AT and use them directly.